package.json

```diff
@@ -6,24 +6,29 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "lint": "eslint . --report-unused-disable-directives --max-warnings 0",
```

Comment:
If we're adding eslint, we should probably add prettier too at the same time.

Reply:
I didnt add anything, I just updated it. I am more than happy to include Prettier too though!

Reply:
Sorry, I saw the new file and missed that you were just moving it.

Reply:
Still happy to add Prettier if you like 😆

Reply:
Yes, let's try and add prettier to all of these repos. It will keep diffs cleaner.